src/model/CustomMenuHolder.php

```diff
@@ -89,6 +89,8 @@ public function getCMSFields()
     function requireDefaultRecords()
     {
         parent::requireDefaultRecords();
+        $config = SiteConfig::current_site_config();
+        $menus = CustomMenuHolder::get();
 
         // Main Menu
         if ($this->class == 'CustomMenuHolder') {
```

Comment:
This needs to be updated to `::class`

Reply:
Also, the line  `if (!DataObject::get_one($this->class)) {` can probably be replaced with `$menus->exists()`. In fact, if you can probably merge the two if statements into one line?